docs/src/content/docs/fr/resources/plugins.mdx

```diff
@@ -68,6 +68,11 @@ Les [modules d'extension](/fr/reference/plugins/) peuvent personnaliser la confi
 		title="starlight-utils"
 		description="Étendre Starlight avec une collection d'utilitaires courants."
 	/>
+	<LinkCard
+		href="https://github.com/trueberryless/starlight-view-modes"
+		title="starlight-view-modes"
+		description="Ajoutez différents modes d'affichage à votre site web de documentation."
```

Comment:
```suggestion
		description="Ajouter différents modes d'affichage à votre site web de documentation."
```

Just to match the other plugins wording but other than that, looks perfect to me 👍 

Thanks for the update 🙌